test/flink_test.go

```diff
@@ -155,11 +155,48 @@ func (s *CLITestSuite) TestFlink_Autocomplete() {
 
 func (s *CLITestSuite) TestFlinkShell() {
 	tests := []FlinkShellTest{
-		{inputFile: "use-catalog", goldenFile: "use-catalog.golden"},
-		{inputFile: "use-database", goldenFile: "use-database.golden"},
-		{inputFile: "set-key", goldenFile: "set-key.golden"},
-		{inputFile: "reset-key", goldenFile: "reset-key.golden"},
-		{inputFile: "reset-all", goldenFile: "reset-all.golden"},
+		{
+			name:       "TestUseCatalog",
+			goldenFile: "use-catalog.golden",
+			commands: []string{
+				"use catalog default;",
+				"set;",
+			},
+		},
+		{
+			name:       "TestUseDatabase",
+			goldenFile: "use-database.golden",
+			commands: []string{
+				"use db1;",
+				"set;",
+			},
+		},
+		{
+			name:       "TestSetSingleKey",
+			goldenFile: "set-key.golden",
```

Comment:
Another great improvement might be to let the golden files describe the test, instead of `name`. This'll make it much quicker to write new tests.
```suggestion
			goldenFile: "set-single-key.golden",
```

Reply:
done